deeppavlov/skills/default_skill/default_skill.py

```diff
@@ -0,0 +1,77 @@
+from deeppavlov.core.models.component import Component
+from deeppavlov.core.skill.skill import Skill
+
+EXPECTING_ARG_MESSAGE = 'expecting_arg:{}'
+
+
+class DefaultStatelessSkill(Skill):
+    """Default stateless skill class.
+
+    The class is intended to be used for as a default skill wrapping
+    DeepPavlov models.
+
+    Attributes:
+        model: DeepPavlov model to be wrapped into default skill instance.
+    """
+    def __init__(self, model: Component):
+        self.model: Component = model
+
+    def __call__(self, utterances_batch: [list, tuple], history_batch: [list, tuple],
+                 states_batch: [list, tuple] = None) -> (list, list, list):
```

Comment:
`(list, list, list)` is a shorthand for `Union[list, list, list]`
You should use `Tuple[list, list, list]`

Reply:
Fixed.